manifests/crds/core.observatorium.io_observatoria.yaml

```diff
@@ -1321,6 +1329,10 @@ spec:
                       type: integer
                     description: Loki replicas per component
                     type: object
+                  serviceMonitor:
+                    description: ServiceMonitor enables  deploying a service monitor
```

Comment:
We need to regenerate this CRD now. Funny, i think CI should really catch that this file is out of date

Reply:
I regenerated this CRD. That is why it can pass CI.

Reply:
I found there is an redundant space here. Let me update it.